samples/js/photos/src/photo-grid-item.js

```diff
@@ -56,6 +58,7 @@ export default class PhotoGridItem extends React.Component<void, Props, State> {
     };
     this._parentTop = 0;
     this._parentLeft = 0;
+    this._unmounted = false;
```

Comment:
Shouldn't this be true?

I would use a mounted variable and set it to true on componentDidMount, etc.